### PR TITLE
7 taxonomy summaries

### PR DESCRIPTION
**Pull request by wasade**
depends on #166 and #167

### CODE REVIEW

**Comment by wasade** [comment]
@mortonjt @antgonza @josenavas @ElDeveloper @squirrelo anyone able to review?

File: ipynb/primary-processing/7-taxonomy-summaries.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+In order to provide taxonomy information for participants, we need to summarize the taxonomies of the individual OTUs. Within AG and for participants, were interested in providing information at 3 levels: phylum, class and genus. These summarizations are used in different areas of the participant results.
```
Comment (squirrelo) [inline ipynb/primary-processing/7-taxonomy-summaries.md]: were -> we're

**Comment by squirrelo** [comment]
One small text change, but :+1: otherwise

**Comment by wasade** [comment]
Thanks!

**Comment by wasade** [comment]
good to go?

**Pull request merged**